app/Filament/Resources/ResultResource.php

```diff
@@ -316,9 +319,19 @@ public static function table(Table $table): Table
                 Tables\Columns\IconColumn::make('healthy')
                     ->boolean()
                     ->toggleable()
-                    ->toggledHiddenByDefault()
+                    ->when(
```

Comment:
Let's leave the current functionality for the healthy column. Other than that looks good to me.